pom.xml

```diff
@@ -7,12 +7,17 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.455</version>
+    <version>1.596.1</version>
   </parent>
   <artifactId>log-parser</artifactId>
   <packaging>hpi</packaging>
   <version>1.1-SNAPSHOT</version>
   <name>Log Parser Plugin</name>
+
+  <properties>
+    <workflow-jenkins-plugin.version>1.4</workflow-jenkins-plugin.version>
```

Comment:
Seeing as the version is only ref'd in one place, is it really worth splitting out a property for it?